index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */ 
+function appear() {
+  const readMoreButton = document.getElementById('read-more-button');
+  const additionalSingers = document.querySelectorAll('.additional-singers-article');
+
+  // var arrowForward = document.getElementById('arrowForward');
+  // var arrowBackward = document.getElementById('arrowBackward').innerHTML('22');
```

Comment:
- [ ] Please remove all commented code from your codebase to make it easy to read and maintain